### PR TITLE
Repeated scalar (and enum) fields should be read-only

### PR DESCRIPTION
**Pull request by nelfin**
Repeated fields do not support assignment regardless if they are composite or not.

Given `repro.proto`:
```protobuf
syntax = "proto2";

message RepeatedFields {
  enum InnerEnum {
    VAL = 0;
  }
  repeated string strs = 1;
  repeated InnerEnum vals = 2;
}
```

```python
>>> from repro_pb2 import RepeatedFields
>>> r1 = RepeatedFields()
>>> r2 = RepeatedFields()
>>> r1.strs = r2.strs
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: Assignment not allowed to repeated field "strs" in protocol message object.
>>> r1.vals = r2.vals
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: Assignment not allowed to repeated field "vals" in protocol message object.
```

### CODE REVIEW

**Comment by nelfin** [comment]
I've also signed the CLA available at https://opensource.dropbox.com/cla/individual/

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 - Organized generated enum code to prevent definition ordering issues in Pyright-based linters
 - Changed type generation for `grpcio` stubs to use the `MultiCallable` API ([see here](https://grpc.github.io/grpc/python/grpc.html#multi-callable-interfaces)) . This requires using the `grpc-stubs` typings for grpcio. This change should allow calling stub methods with common parameters (`timeout`, `metadata`, etc.) as well as calling methods on the `MultiCallable` object (e.g. `my_stub.MyRpcMethod.future()`).
+- Update stubs to mark repeated scalar and enum fields as read-only
```
Comment (nipunn1313) [inline CHANGELOG.md:5]: minor grammatical clarification. Could you update to say
```
repeated scalar and repeated enum fields as read-only
```

it's ambiguous whether "repeated" applies to enum as written.

File: README.md [hunk 2]
```diff
@@ -197,6 +197,7 @@ black mypy_protobuf/main.py test/
 - [@henribru](https://github.com/henribru)
 - [@Evgenus](https://github.com/Evgenus)
 - [@MHDante](https://github.com/MHDante)
+- [@nelfin](https://github.com/nelfin)
```
Comment (nipunn1313) [inline README.md:200]: ❤️

**Comment by nipunn1313** [comment]
Thank you so much! This looks great. One minor comment regarding release notes - and then I'll merge it.

File: CHANGELOG.md (hunk 1 above)
Comment (nelfin) [reply CHANGELOG.md]: Done

**Comment by nipunn1313** [comment]
BAM awesome

**Pull request merged**